.github/workflows/ci.yml

```diff
@@ -1,8 +1,26 @@
-name: Cypress Tests
+name: Coderadio-client ci
 
 on: [push, pull_request]
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-16.04
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout Source Files
+        uses: actions/checkout@v2
+
+      - name: Install modules
+        run: yarn
+
+      - name: Run ESLint
+        run: yarn run eslint . --ext .js,.jsx,.ts,.tsx
```

Comment:
Thank you, for this PR @Ismailtlem . I just wonder if we should use `yarn` considering it is not used in the app itself.

Reply:
Corrected